### PR TITLE
feat(documentation): Add documentation for non-obvious bits

### PR DESCRIPTION
**Pull request by ghost**
Hi guys, please check out the docs for the most interesting parts of FueledUtils. Any feedback would be highly appreciated. Thanks!

### CODE REVIEW

File: FueledUtils/HairlineView.swift [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import Foundation
 import UIKit
 
+/// A view with intrinsic content size if 1px by 1px
```
Comment (benoit-fueled) [inline FueledUtils/HairlineView.swift]: Little typo, `of` I guess

File: FueledUtils/KeyboardInsetHelper.swift [hunk 2]
```diff
@@ -1,10 +1,18 @@
 import Foundation
 import UIKit
 
+/// Binds keyboard appearance and metrics to scroll view content and scroll bar insets and/or a layout constraint \
+/// relative to reference view. This object is can to be created and linked in a sotryboard.
```
Comment (benoit-fueled) [inline FueledUtils/KeyboardInsetHelper.swift]: Little typos here too I think. `This object can be created and linked in a storyboard.` ?

**Comment by ritesh-fueled** [comment]
looks good @leonty-fueled other than the typos pointed out by Benoit 👌Also if time permits, how about we prepare some wiki docs (or for now may be just a big Readme) as well where we could add some examples highlighting the usage?

File: FueledUtils/KeyboardInsetHelper.swift (hunk 2 above)
Comment (ghost) [reply FueledUtils/KeyboardInsetHelper.swift]: LOL, good catch! 😂

**Comment by ghost** [comment]
Good idea @ritesh-fueled! We might want to think more of first-time adoption experience. Maybe some index of problems that the library solves.

**Comment by ghost** [comment]
Looks good to me, it does help comprehend better what these components are supposed to be doing

**Pull request merged**